src/components/TopApis.tsx

```diff
@@ -0,0 +1,85 @@
+"use client"; // useState を使うために必要
+
+import React, { useState } from "react";
+
```

Comment:
eg. using defined style w/tailwind.css 
```suggestion
// import css file
import { TopApisStyle } from "@/src/style/style.css"

// in style.css
const TopApisStyle = {
  tabButton: {
    height: 30px,
    width: 60px,
    backgroud-color: black,
  },
} 

// use style
<Button className={TopApisStyle.tabButton}>
  <p>xxx</p>
</Button>
```